src/GlobalX.ChatBots.WebexTeams/Services/WebexTeamsMessageHandler.cs

```diff
@@ -21,8 +23,33 @@ internal class WebexTeamsMessageHandler : IMessageHandler
 
         public async Task<Message> SendMessageAsync(Message message)
         {
+            var request = _messageParser.ParseCreateMessageRequest(message);
+
+            Models.Message result;
+            try
+            {
+                result = await _apiService.SendMessageAsync(request).ConfigureAwait(false);
+            }
+            catch (InvalidParentException)
```

Comment:
Is there a way to catch this before we try the request? Just to avoid the overhead of two requests

Reply:
Unfortunately, not really.

Well, we *could* check the value of `message.ParentId` and do a pre-emptive `GetMessage` on it, to check if *that* has a `parentId`, and if so, use it. But then we'd be making two requests *every* time instead of just if the comment being replied to is inline.

Unfortunately Webex doesn't have any other way that I could find to determine if a comment is top-level or inline.